# State general
robotnik_msgs/State state
# JOINTBYJOINT, CARTESIAN-EULER
string control_type
# current group
string current_group
# Selected joint
string selected_joint
# Current velocity level value
float32 velocity_level
# Shows if deadman is pressed
bool deadman_active
